Mono.Debugging.Soft/SoftDebuggerSession.cs

```diff
@@ -2850,25 +2859,18 @@ bool CheckBetterMatch (TypeMirror type, string file, int line, int column, Locat
 							return true;
 						if (entry.Row == line && column >= entry.Column && entry.Column > found.ColumnNumber)
 							return true;
-						if (vm.Version.AtLeast (2, 19)) { //if version is less then 2.19, found Location will not contain info about columns
```

Comment:
This was a request from @DavidKarlas. The previous line is identical to the one I removed, except for the vm check. The end result is that if the vm version check passes, the other will fail every time, so that whole thing was not needed. I can separate that check into a different commit for tracking purposes.